test/test_reads.py

```diff
@@ -819,6 +820,99 @@ def testTranslationOfStopCodonUAA(self):
             next(read.translations()))
 
 
+class TestDNAKozakRead(TestCase):
+    """
+    Test the DNAKozakRead class.
+    """
+    def testSequence(self):
+        """
+        A DNAKozakRead instance must have the correct sequence.
+        """
+        originalRead = DNARead('id', 'AAGTAAGGGCTGTGA')
+        read = DNAKozakRead(originalRead, 0, 4, 100.0)
+        self.assertEqual('AAGT', read.sequence)
+
+    def testStart(self):
+        """
+        A DNAKozakRead instance must store the correct start offset.
+        """
+        originalRead = DNARead('id', 'AAGTAAGGGCTGTGA')
+        read = DNAKozakRead(originalRead, 2, 4, 100.0)
+        self.assertEqual(2, read.start)
+        read = DNAKozakRead(originalRead, 4, 10, 100.0)
```

Comment:
Unless there's a point to testing the same thing twice, don't. It just makes more code and it makes your tests less clear (the reader will wonder if they're missing something: "uh, didn't she just test that, what am I missing here?").

Reply:
done